content/building/build-nxp.md

```diff
@@ -100,7 +100,7 @@ After cloning the repo, you need to setup the build environment. You can use the
 
 ### Automated Install of the build environment
 
-__The following power shell script is not signed. Run Power Shell as an Administrator and run `set-executionpolicy remotesigned` to enable execution of the non-signed script.__
+__The following power shell script is not signed. Run Power Shell as an Administrator and run `set-executionpolicy Unrestricted` to enable execution of the non-signed script.__
```

Comment:
@josesimoes Can we make this remote signed now we have a code cert? otherwise this change would be correct...

Reply:
@microDevSys Revert this change. After internal discussion, it is an issue that we are not signing our powershell scripts. I will create an issue for it.